src/cpp/rtps/resources/DeadlineTimer.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
I think best location for this file is `src/cpp/timedevent`

Reply:
Moved to a new location ```src/cpp/timedevent```, see commit https://github.com/eProsima/Fast-RTPS/commit/2c459e3d1222438e87626e241c4b8be697b487af